tests/testthat/test-build-favicons.R

```diff
@@ -1,9 +1,8 @@
 test_that("missing logo generates message", {
   pkg <- local_pkgdown_site(test_path("assets/site-empty"))
 
-  expect_error(
-    expect_snapshot(build_favicons(pkg)),
-    "Can't find package logo"
+  expect_snapshot(error = TRUE,
+    expect_output(build_favicons(pkg), "Building favicons")
```

Comment:
For some reasons, `cli_alert_*()` are not shown in snapshot tests, hence the `expect_output()` here to capture the desired behavior. As you can see in the previous snapshot, there was nothing.

Reply:
Hmmm. We should probably switch those to `cli_inform()`, just for consistency, but that can be a separate PR.